packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -1265,15 +1279,20 @@ export class FluidDataStoreRuntime
 				channelContext.rollback(envelope.contents, localOpMetadata);
 				break;
 			}
-			default:
+			default: {
 				throw new LoggingError(`Can't rollback ${type} message`);
+			}
 		}
 	}
 
+	// TODO: use something other than `any` here
+	// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
 	public async applyStashedOp(content: any): Promise<unknown> {
```

Comment:
This method accesses properties on `content`, but I have no idea what type we assume it is. Leaving this for someone else to fix.